frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -64,6 +64,14 @@ export const getOCSVersion = (items: FirehoseResult): string => {
   return _.get(operator, 'status.installedCSV');
 };
 
+export const getLSOVersion = (operator: K8sResourceKind): string => {
+  return _.get(operator, 'status.installedCSV');
+};
+
+export const getLSONamespace = (operator: K8sResourceKind): string => {
+  return _.get(operator, 'metadata.namespace');
+};
```

Comment:
```suggestion
```

Reply:
thanks for the review... I have made the required changes...